plugins/drivers/testutils/testing.go

```diff
@@ -75,6 +75,41 @@ func (h *DriverHarness) Kill() {
 	h.server.Stop()
 }
 
+// tinyChroot is useful for testing, where we do not use anything other than
+// trivial /bin commands like sleep and sh.
+//
+// Note that you cannot chroot a symlink.
+var tinyChroot = map[string]string{
+	// destination: /bin
+	"/usr/bin/sleep": "/bin/sleep",
+	"/usr/bin/dash":  "/bin/sh",
```

Comment:
This is Ubuntu/Debian specific IIRC. Is that going to be fragile?

Reply:
Yeah, but this isn't the only ubuntu-ism we have in testing - e.g. a number of error strings reported by the OS are explicitly checked too. 

I played around with way to dynamically get these, but it doesn't necessarily get what you want
```
+func find(t testing.T, program string) string {
+       // find program on $PATH
+       link, whichErr := exec.Command("which", program).CombinedOutput()
+       require.NoError(t, whichErr)
+       require.NotEmpty(t, link, "could not find %q", program)
+       // resolve program if it is a link
+       executable, err := exec.Command("readlink", "-f", string(link)).CombinedOutput()
+       require.NoError(t, err)
+       require.NotEmpty(t, executable, "could not resolve link %q", link)
+       return strings.TrimSpace(string(executable))
+}
```

We can iterate on that down the line if we find a problem